atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/CassandraTimestampBoundStore.java

```diff
@@ -142,6 +149,7 @@ private void cas(Client client, Long oldVal, long newVal) {
             lastWriteException = err;
```

Comment:
using above, something along the lines of:

```
        if (!result.isSuccess()) {
            String msg = "Timestamp limit changed underneath us (limit in memory: " + currentLimit
                    + "). This may indicate that another timestamp service is running against this cassandra keyspace."
                    + " This is likely caused by multiple copies of a service running without a configured set of"
                    + " leaders or a CLI being run with an embedded timestamp service against an already running"
                    + " service.\n"
                    + "Current values: " + result.getCurrent_values().stream()
                            .map(c ->
                                    "{column=" + new String(c.getName(), Charsets.UTF_8)
                                            + ", value=" + BaseEncoding.base16().encode(c.getValue())
                                            + ", C* timestamp=" + c.getTimestamp()
                            )
                            .collect(Collectors.toList());
            Error err = exit(msg);
            lastWriteException = err;
            throw err;
```